src/world.jl

```diff
@@ -3,7 +3,12 @@ include("lib/datastore.jl")
 
 suffix = getsuffix()
 
-mastercounties = readtable(datapath("global/counties$suffix.csv"), eltypes=[UTF8String, UTF8String, UTF8String])
+if config["netset"] == "three"
+	mastercounties = readtable(datapath("global/counties$suffix.csv"), eltypes=[UTF8String, UTF8String, UTF8String])
+else
+	mastercounties = readtable(datapath("global/counties.csv"), eltypes=[UTF8String, UTF8String, UTF8String])
+end
+
```

Comment:
Yeah, let's keep it as you have it for now.  We can come up with a more general solution as we encounter more case-studies.